src/emcpy/plots/create_plots.py

```diff
@@ -390,7 +390,7 @@ def _map_scatter(self, plotobj, ax):
 
             cs = ax.scatter(plotobj.longitude, plotobj.latitude,
                             s=plotobj.markersize, **inputs,
-                            transform=self.projection.projection)
+                            transform=self.projection.transform)
```

Comment:
want @kevindougherty-noaa to confirm this still works as intended with this change

Reply:
Oh I see what this is doing now, it should be fine.

Reply:
Yes, transform for NP/SP stereographic projections have to be PlateCarree().